node-bootstrapper/main.go

```diff
@@ -158,9 +189,27 @@ func provisionStart(ctx context.Context, config *datamodel.NodeBootstrappingConf
 	if err != nil {
 		return fmt.Errorf("preparing CSE: %w", err)
 	}
-	// TODO: add Windows support
-	//nolint:gosec // we generate the script, so it's safe to execute
-	cmd := exec.CommandContext(ctx, "/bin/bash", "-c", cse.UnsafeValue())
+
+	var cmd *exec.Cmd
+	if config.AgentPoolProfile.IsWindows() {
+		systemDrive := os.Getenv("SYSTEMDRIVE")
+		slog.Info(fmt.Sprintf("systemDrive: %s\n\n", systemDrive))
+		if systemDrive == "" {
+			systemDrive = "C:"
+		}
+		script := string(cse)
+		script = strings.ReplaceAll(script, "%SYSTEMDRIVE%", systemDrive)
```

Comment:
You can check https://pkg.go.dev/os#ExpandEnv for an inspiration (seems like it's not windows friendly).

Do you want to envsubst a single variable or all of them?

Reply:
Just the one needed to be substituted. Will have a look.